xc-tool/src/evm.rs

```diff
@@ -0,0 +1,13 @@
+use crate::util::ErrBox;
+
+use ethers::prelude::*;
+
+use ethers::providers::Http;
+
+/// Query the Pyth contract's configuration from an EVM chain
```

Comment:
Just curious, how are you testing this?

Reply:
Unfortunately, it doesn't even build because the codegen tries to `derive(Debug)` on a huge-ass tuple which the standard library does not support (They literally just implement `Debug` statically for a bunch of tuple types between 1 and 9 members that are already `Debug`)